chia/cmds/plotnft_funcs.py

```diff
@@ -151,16 +154,20 @@ async def pprint_pool_wallet_state(
 
 
 async def show(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
-
     config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
     self_hostname = config["self_hostname"]
     farmer_rpc_port = config["farmer"]["rpc_port"]
     farmer_client = await FarmerRpcClient.create(self_hostname, uint16(farmer_rpc_port), DEFAULT_ROOT_PATH, config)
     address_prefix = config["network_overrides"]["config"][config["selected_network"]]["address_prefix"]
     summaries_response = await wallet_client.get_wallets()
     wallet_id_passed_in = args.get("id", None)
+    plot_counts: Counter = Counter()
     try:
         pool_state_list: List = (await farmer_client.get_pool_state())["pool_state"]
+        harvesters = await farmer_client.get_harvesters()
+        for d in harvesters["harvesters"]:
+            for plot in d["plots"]:
+                plot_counts[hexstr_to_bytes(plot["pool_contract_puzzle_hash"])] += 1
```

Comment:
@emlowe Good point, should probably do something like below as `pool_contract_puzzle_hash` is always in there but its `None` for OG plots which will cause `hexstr_to_bytes` to fail. 
```suggestion
                if plot["pool_contract_puzzle_hash"] is not None:
                    plot_counts[hexstr_to_bytes(plot["pool_contract_puzzle_hash"])] += 1
```